workflows/sarek.nf

```diff
@@ -335,10 +335,13 @@ workflow SAREK {
     bwa                    = params.bwa                     ? Channel.fromPath(params.bwa).collect()       : PREPARE_GENOME.out.bwa
     bwamem2                = params.bwamem2                 ? Channel.fromPath(params.bwamem2).collect()   : PREPARE_GENOME.out.bwamem2
     dragmap                = params.dragmap                 ? Channel.fromPath(params.dragmap).collect()   : PREPARE_GENOME.out.hashtable
+    sentieon               = params.sentieon                ? Channel.fromPath(params.sentieon).collect()  : PREPARE_GENOME.out.sentieon
```

Comment:
we should use params.bwa for that, the way I see it params.sentieon should only be a flag to allow usage of aligner sentieon-bwamem and the extra sentieon tools

Reply:
agreed

Reply:
basically select that you want to use `bwa` but the `sentieon` implementaion

Reply:
but for that I would specify `--aligner sentieon-bwamem`